src/bin/cql-stress-cassandra-stress/settings/param/multi_param.rs

```diff
@@ -0,0 +1,240 @@
+use std::{cell::RefCell, collections::HashMap, rc::Rc};
+
+use anyhow::Result;
+use regex::Regex;
+
+use super::{Param, ParamCell, ParamHandle, ParamMatchResult};
+
+lazy_static! {
+    // Multiparameters must match the pattern:
+    // param_name(key1=value1,somethingThatMatchesPredefinedSubparam,key2=value2,...keyN=valueN)
+    // These regular expressions help to validate and extract the subparameters from the string.
+    static ref SUBPARAM: Regex = Regex::new(r"([^,]+)").unwrap();
+    static ref ARBITRARY_PARAM: Regex = Regex::new(r"^([^=]+)=([^=]+)$").unwrap();
```

Comment:
Just a question: why is `ARBITRARY_PARAM` regex a `^ ... $` regex while `SUBPARAM` is not?

Reply:
To give an example:
Assume we receive `foo=bar,key=value,factor=3`. Firstly, the `SUBPARAM` regexp is used to iterate over substrings that match this pattern i.e. `foo=bar`, `key=value` and `something=something`. Note that we cannot wrap the regexp with `^$`, because we want to access multiple substrings and iterate over them (if we contained `^` and `$` we would get no match - so nothing to iterate over).

Now that we extracted three subparameters, we firstly try to parse them as some predefined subparameter (in `replication`'s case it's for example `factor` subparam). If it didn't match any of the prefedined subparameters, we try to parse it as an arbitrary subparameter (something that exactly matches `ARBITRARY_PARAM` pattern).

I hope this answers your question.

Reply:
Yes, it makes sense. If I understand it right, your solution permits multiple ','? Like: `replication(foo=bar,,,,key=value)`. It might have been easier to just split the string on ',' instead of using the regex.

Reply:
Now that I think of that, I should add some comment/more descriptive name for `SUBPARAM`. At first glance, it's not clear that it's used to extract the sub parameters delimited by a comma.

Reply:
> Yes, it makes sense. If I understand it right, your solution permits multiple `,`? Like: `replication(foo=bar,,,,key=value)`. It might have been easier to just split the string on `,` instead of using the regex.

Oh, you are right. And yes, the split would make more sense here.